sdk/eventgrid/Microsoft.Azure.Messaging.EventGrid.CloudNativeCloudEvents/src/Microsoft.Azure.Messaging.EventGrid.CloudNativeCloudEvents.csproj

```diff
@@ -20,6 +20,6 @@
     <PackageReference Include="CloudNative.CloudEvents.SystemTextJson" />
 <!-- This is required to prevent version conflict stemming from The CloudNative.CloudEvents.SystemTestJson depending on -->
```

Comment:
```suggestion
  </ItemGroup>

  <ItemGroup Condition="'$(TargetFramework)' != 'net6.0'">
<!-- This is required to prevent version conflict stemming from The CloudNative.CloudEvents.SystemTestJson depending on -->
```